cocos2d/CCScene+Private.h

```diff
@@ -1,9 +1,7 @@
 /*
```

Comment:
Speaking of which, CCScene+Private.h should be CCScene_Private.h to be consistent. Will fix.